docs/src/Re-use-Martin-as-a-library.md

```diff
@@ -0,0 +1,9 @@
+# Re-use Martin as a library
```

Comment:
this should be added to the index.  Perhaps under `For developers` section?  There we should also add things like how to contribute to Martin

Reply:
Oops. Forget to add it to the index.    
And any thounghts on the developers section? A new `For developers` section?  Perhaps place these things to the old `Development` section?

```markdown
- [Development](development.md)
 - [Prerequisites](Prerequisites.md) things about the `docker` `docker-compose` `just` `jq`  etc
 - [with VS Code](with-VS-Code.md) how to build and debug with vscode and the plugins needed(CODELLDB, rust analyzer.. etc)
 - [With Rust Rover](with-rust-rover.md) have no too much experience on this...
 - [Martin as a library](martin-as-a-library.md)
```